lib/module.js

```diff
@@ -199,3 +244,16 @@ export default function SentryModule (moduleOptions) {
     })
   }
 }
+
+async function getBrowserApiMethods () {
+  const SentryBrowser = await import('@sentry/browser')
+
+  const browserMethods = []
+  for (const key in SentryBrowser) {
```

Comment:
Just a style thing but iteration using `const [key, value] of Object.entries(SentryBrowser)` would be nicer.

Reply:
I have a love/hate relationship with Object.entries and in this case it feels way to verbose, it wont make the code better readable at all imo